README.md

```diff
@@ -186,95 +217,93 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Question:** Describe what is happening in the code above.
 
-* **Your Answer:**
+* **Your Answer:** We will only show '/users' route if there is a currentUserId.
 
 ---
 
-- [ ] Now try logging in. Then, when you're on the `/users` page, refresh the page.
+- [√] Now try logging in. Then, when you're on the `/users` page, refresh the page.
 
 * **Question:** What happens and why?
 
-* **Your Answer:**
+* **Your Answer:** When we refresh, we are redirected to '/login' page.
 
 ---
 
-- [ ] To solve this problem, let's add a `loading` key to our App's state, with the default value set to `true`. When `componentDidMount()` finishes, set the `loading` key to equal `false`. Using this key, solve the issue of refreshing on the `/users` page. Make sure everyting continues to work whether you are logged in or out.
+- [√] To solve this problem, let's add a `loading` key to our App's state, with the default value set to `true`. When `componentDidMount()` finishes, set the `loading` key to equal `false`. Using this key, solve the issue of refreshing on the `/users` page. Make sure everyting continues to work whether you are logged in or out.
 
 * **Question:** What did you do to solve this problem?
 
-* **Your Answer:**
+* **Your Answer:** We set the default state to be 'loading: true' and then in componentDidMount, we changed the loading state to be false. We then specified in the render that if the loading state was 'true' to display a <p>.
 
 ---
 
 - [ ] We will have the same problem on the `/users/<userId>/posts` page. Use the same strategy to have this page load correctly on refresh.
 
 * **Question:** In what component did you add the `loading` property and why?
 
-* **Your Answer:**
+* **Your Answer:** We skipped this in class and there didnt seem to be a need for it.
```

Comment:
If you add the loading property in app then the entire site will hang until you are verified but you only need to have it in one place. you may decide to conditionally render different components depending on wether they should be restricted or not.